Sprint-1/exercises/paths.js

```diff
@@ -16,3 +16,7 @@ console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
 // Create a variable to store the ext part of the variable
+
+const dir=filePath.slice(0, lastSlashIndex).slice(1);
```

Comment:
dir includes the root, so you don't have to get rid of the first '/' character here.

Reply:
I have removed slice to include the root directory . so now dir value will include root path